ApplicationPattern+data.json

```diff
@@ -23,7 +23,7 @@
                 },
                 "action-profile-configuration": {
                   "request": {
-                    "protocol-reference": "/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/protocol",
+                    "protocol-reference": "/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol",
```

Comment:
we have uuid in profile section starting with prefix * and xx, which will be replaced by the actual application-name's notation in individual application. Here, uuid prefix is * in profile section whereas xx in logical-termination-point list. please let me know if we can maintain a common pattern for denoting the application-name holder (either * or xx) to maintain hormony.

Reply:
Made a CTRL+h from *-1-0-0 to xx-1-0-0. Thanks for indicating.